x-pack/plugins/data_visualizer/common/constants.ts

```diff
@@ -56,3 +63,116 @@ export const featureTitle = i18n.translate('xpack.dataVisualizer.title', {
+              }
+            ) +
+            '</a>',
+        },
+      });
+    case SUPPORTED_FIELD_TYPES.GEO_POINT:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.geoPointField', {
+        defaultMessage: 'Latitude and longitude points.',
+      });
+    case SUPPORTED_FIELD_TYPES.GEO_SHAPE:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.geoShapeField', {
+        defaultMessage: 'Complex shapes, such as polygons.',
+      });
+    case SUPPORTED_FIELD_TYPES.HISTOGRAM:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.histogramField', {
+        defaultMessage: 'Pre-aggregated numerical values in the form of a histogram.',
+      });
+    case SUPPORTED_FIELD_TYPES.IP:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.ipAddressField', {
+        defaultMessage: 'IPv4 and IPv6 addresses.',
+      });
+    case SUPPORTED_FIELD_TYPES.IP_RANGE:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.ipAddressRangeField', {
+        defaultMessage: 'Range of ip values supporting either IPv4 or IPv6 (or mixed) addresses.',
+      });
+    case SUPPORTED_FIELD_TYPES.MURMUR3:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.murmur3Field', {
+        defaultMessage: 'Field that computes and stores hashes of values.',
+      });
+    case SUPPORTED_FIELD_TYPES.NESTED:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.nestedField', {
+        defaultMessage: 'JSON object that preserves the relationship between its subfields.',
+      });
+    case SUPPORTED_FIELD_TYPES.NUMBER:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.numberField', {
+        defaultMessage: 'Long, integer, short, byte, double, and float values.',
+      });
+    case SUPPORTED_FIELD_TYPES.STRING:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.stringField', {
+        defaultMessage: 'Full text such as the body of an email or a product description.',
+      });
+    case SUPPORTED_FIELD_TYPES.TEXT:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.textField', {
+        defaultMessage: 'Full text such as the body of an email or a product description.',
+      });
+    case SUPPORTED_FIELD_TYPES.KEYWORD:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.keywordField', {
+        defaultMessage:
+          'Structured content such as an ID, email address, hostname, status code, or tag.',
+      });
+    case SUPPORTED_FIELD_TYPES.VERSION:
+      return i18n.translate('xpack.dataVisualizer.index.fieldNameDescription.versionField', {
+        defaultMessage: 'Software versions. Supports {SemanticVersioningLink} precedence rules.',
+        values: {
+          SemanticVersioningLink:
+            `<a href="https://semver.org/"
```

Comment:
Any reason this has to be a string of an a tag as opposed to a link component from the eui library?

Reply:
This is originally copied from Discover. Since this is in a constant file used in a helper function, using EuiLink would require this file to be importing React. However, if readability is important, we can definitely switch over.

Reply:
Sounds good 👍